doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+   "id": "35b5e132-8fde-4637-8289-e29bc0bb4cb5",
+   "metadata": {},
+   "source": [
+    "(option-cnorm)=\n",
+    "## `cnorm`\n",
+    "\n",
+    "The `cnorm` option controls how data values are mapped to colors in a colormap. It affects the distribution of colors across the range of values.\n",
+    "\n",
+    "Accepted values:\n",
+    "- 'linear' (default): evenly maps values across the colormap.\n",
+    "- 'log': applies logarithmic scaling, useful for data with large dynamic range.\n",
+    "- 'eq_hist': uses histogram equalization to emphasize contrast in sparse or skewed data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "18379de6-42dd-4e1d-9a6c-740c6fe226a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='depth',\n",
+    "    cmap='inferno_r',\n",
+    "    bgcolor='#f5f5f5',\n",
+    "    cnorm='eq_hist',\n",
+    "    title=\"Earthquakes with Histogram Equalization color scaling\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7dc82357-e05f-4313-9599-8b7eeb0ac2ff",
+   "metadata": {},
+   "source": [
+    ":::{tip}\n",
+    "Change the `cnorm` value to 'log' and then 'linear' to see how the color distribution changes\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "881751c3-4dc2-4027-9b7f-147fbe64a4f6",
+   "metadata": {},
+   "source": [
+    "(option-fontscale)=\n",
+    "## `fontscale`\n",
+    "\n",
+    "The `fontscale` option scales all the fonts in the plot by the provided numeric factor. For example, setting `fontscale=1.5` enlarges the title, tick labels, and axis labels by 50%. This is useful when you want to emphasize text for presentations or detailed viewing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1ddd044-1771-43a3-a09d-580749371644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontscale=1.5,\n",
+    "    title=\"Penguins Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee479a25-5270-4b60-91c3-5da24ae2a501",
+   "metadata": {},
+   "source": [
+    "(option-fontsize)=\n",
+    "## `fontsize`\n",
+    "\n",
+    "The `fontsize` option sets the font size for different text elements in the plot. It can be:\n",
+    "- A single value (e.g. 12, '14pt', or '10px') to apply to all text.\n",
+    "- A dictionary to control individual elements like title, axes labels, or ticks.\n",
+    "Example: {'title': '15pt', 'xlabel': '12pt', 'ylabel': '12pt', 'ticks': 10}\n",
```

Comment:
Hm that looks very Bokeh specific. Does it work too with Matplotlib? I'd suggest adding a Matplotlib example here .